examples/gallery/histograms/scatter_and_histograms.py

```diff
@@ -2,47 +2,48 @@
 Scatter plot with histograms
 ============================
 
-To create a scatter plot with histograms at the sides of the plot one
-can use :meth:`pygmt.Figure.plot` in combination with
-:meth:`pygmt.Figure.histogram`. The positions of the histograms are plotted
-by offsetting them from the main scatter plot figure using
-:meth:`pygmt.Figure.shift_origin`.
+To create a scatter plot with histograms at the sides of the plot one can use
+:meth:`pygmt.Figure.plot` in combination with :meth:`pygmt.Figure.histogram`.
+The positions of the histograms are plotted by offsetting them from the main
+scatter plot using :meth:`pygmt.Figure.shift_origin`.
 """
 
 # %%
 import numpy as np
 import pygmt
 
-# Generate random data from a standard normal distribution centered on 0
-# with a standard deviation of 1
+# Generate random data from a standard normal distribution centered on 0 with a
+# standard deviation of 1
 rng = np.random.default_rng(seed=19680801)
 x = rng.normal(loc=0, scale=1, size=1000)
 y = rng.normal(loc=0, scale=1, size=1000)
 
 # Get axis limits
 xymax = max(np.max(np.abs(x)), np.max(np.abs(y)))
```

Comment:
Currently, the example only works when x and y are in the same range ([-4, 4]) and are centering on 0.0. I think the example will be more useful if we can make it more general, for example, having x in [-4, 4] range and y in [10, 20].

Reply:
Yeah, this was something I also noticed.

It depends until when we want to have the related changes done. Currently I am at a project meeting and I got sick. So, it's not very likely I can do this before AGU.

Ah, BTW - what's @michaelgrund opinion on these changes 🙂, as it's originally his example.

Reply:
Definitely fine for me since it makes the whole example more general.

Reply:
I've updated the example in 468d09220ae6f43c40f7dd46d6356057167254a6.

Reply:
Please see https://github.com/GenericMappingTools/pygmt/pull/3628#pullrequestreview-2459377278.